plugins/inputs/turbostat/README.md

```diff
@@ -0,0 +1,104 @@
+# Turbostat Input Plugin
+
+This plugin monitors system performance using `turbostat`.
+
+⭐ Telegraf v1.35.0
+🏷️ server,system
+💻 linux
+
+## Prerequisites
+
+Install `turbostat`.
+
+Allow the Telegraf user to run `turbostat`. Assuming the Telegraf user is
+`telegraf` and `turbostat` is installed in `/usr/bin`, add the following line
+to `/etc/sudoers.d/telegraf`.
+
+```text
+telegraf ALL=(root) NOPASSWD: /usr/bin/turbostat
+```
+
+## Service Input <!-- @/docs/includes/service_input.md -->
+
+This plugin is a service input. Normal plugins gather metrics determined by the
+interval setting. Service plugins start a service to listens and waits for
+metrics or events to occur. Service plugins have two key differences from
+normal plugins:
+
+1. The global or plugin specific `interval` setting may not apply
+2. The CLI options of `--test`, `--test-wait`, and `--once` may not produce
+   output for this plugin
+
+## Global configuration options <!-- @/docs/includes/plugin_config.md -->
+
+In addition to the plugin-specific configuration settings, plugins support
+additional global and plugin configuration settings. These settings are used to
+modify metrics, tags, and field or create aliases and configure ordering, etc.
+See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
+
+[CONFIGURATION.md]: ../../../docs/CONFIGURATION.md#plugins
+
+## Configuration
+
+```toml
+[[inputs.turbostat]]
+  command = ["sudo", "turbostat", "--quiet", "--interval", "10", "--show", "all"]
+```
+
+Some Turbostat configuration options are particularly relevant for this plugin.
+
+- `-i`, `--interval`: Overrides the default 5s collection interval. This plugin
+is a service input plugin: it ignores the Telegraf global and plugin-specific
+collection intervals. The interval should be configured in Turbostat instead.
+- `-S`, `--summary`: Limits the output to a one-line system summary per
+collection interval.
+- `-s`, `--show`: Show only the specified columns. May be invoked multiple
+times, or with a comma-separated list of column names.
+- `-H`, `--hide`: Do not show the specified columns. May be invoked multiple
+times, or with a comma-separated list of column names. Be careful not to hide
+CPU, Core, Package, and Die, or the output may lose much of its meaning.
+
+To discover the list of available columns, run `sudo tubostat --list`.
+
+For further information, run `man turbostat`. If the man page is not installed,
+you can obtain `tools/power/x86/turbostat/turbostat.8` from the
+[Linux kernel repository](https://github.com/torvalds/linux) and browse it with
```

Comment:
Maybe it would be easier for user if they got a direct download link right away?
https://github.com/torvalds/linux/blob/master/tools/power/x86/turbostat/turbostat.8

Reply:
That's what I initially went for, but I could not get the Markdown linter to stop complaining about excessive line length. Do you know how to do?

Reply:
In my opinion, the linter should be configured not to check the length of the URL.
@srebhan @skartikey @mstrandboge Can you help with this?

Reply:
A reference-style link solves this.